src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -260,15 +260,36 @@ object DemuxFastqs {
       |    --metadata SampleSheet.csv --metrics metrics.txt --output output_folder
       |```
       |
-      |## Illumina FASTQ File Naming
+      |## Output Standards
       |
-      |The output FASTQs will be written according to Illumina naming standards when the `--illumina-standards` option
-      |is specified.  This requires that the input fastq files contain Illumina format read names with comments containing
-      |`<ReadNum>:<FilterFlag>:<0>:<SampleNumber>` (no angle brackets).  The output create output files with Illumina
-      |names:
+      |The following options affect the output format:
       |
-      |1. The file extension will be `_R1_001.fastq.gz` for read one, and `_R2_001.fastq.gz` for read two (if paired end).
-      |2. The per-sample output prefix will be `<SampleName>_S<SampleOrdinal>_L<LaneNumber>` (without angle brackets).
+      |1. If `--fastq-skip-read-numbers` is specified, then trailing /1 and /2 for R1 and R2 respectively, will not be
+      |appended to e FASTQ read name.  By default they will be appended.
+      |2. If `--fastq-include-sample-barcodes` is specified, then sample barcode will replace the last field in the
```

Comment:
Similarly, I think it reads better as `--include-sample-barcodes-in-fastq`

Reply:
Might be nice to include an example here too, to make it clearer - e.g. `@foo.bar.splat beep:boop:ACGTGTA`